api-definitions/README.md

```diff
@@ -0,0 +1,23 @@
+# API Definitions
```

Comment:
Is "API Specification" a better fit?

Reply:
See above comment about `API Definition` vs `API Specification` from Open API docs